### ISSUE TITLE
ABNF: prose values

### ISSUE THREAD

**Issue by mnot** [katef/kgt#10]
ABNF allows a [prose-val](https://tools.ietf.org/html/rfc5234#section-4) to be used:

~~~
         prose-val      =  "<" *(%x20-3D / %x3F-7E) ">"
                                ; bracketed string of SP and VCHAR
                                ;  without angles
                                ; prose description, to be used as
                                ;  last resort
~~~

This is commonly used in IETF RFCs (e.g., for references to rules defined in other specifications). However, it doesn't seem to be supported.

**Comment by katef** [comment]
I'm not sure what to do with these. kgt's aim is to provide transformations of BNF (factoring to LL(1) and other similar sorts of things), as well as to output diagrams as-is.

Perhaps I could store these as a special node, and just error out if you try to output any dialect which doesn't support it, or make transformations using it.

**Comment by katef** [comment]
> 2.1. Rule Naming
>    Unlike original BNF, angle brackets ("<", ">") are not required.  However, angle brackets may be used around a rule name whenever their   presence facilitates in discerning the use of a rule name.  This is   typically restricted to rule name references in free-form prose, or   to distinguish partial rules that combine into a string not separated   by white space, such as shown in the discussion about repetition,   below.

So `<xyz>` is _typically_ prose, but could also be a rule name? Blurgh.

**Comment by katef** [comment]
Implemented in #16.
